### PR TITLE
Update twitter names

### PR DESCRIPTION
**Pull request by epeach**
# Description
Include the twitter handles for the new artists in dance party 2019.

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/code-studio/dancePartySongArtistTags.js [hunk 1]
```diff
@@ -35,13 +35,23 @@ export const SongTitlesToArtistTwitterHandle = {
...
+  calma_pedrocapo: 'Pedrocapo',
+  highhopes_panicatthedisco: 'PanicAtTheDisco',
+  ificanthaveyou_shawnmendes: 'ShawnMendes',
+  introtoshamstep_47SOUL: '47soulofficial',
+  kidzbop_ificanthaveyou_shawnmendes: 'KidzBop',
+  neverreallyover_katyperry: 'KatyPerry',
+  occidentalview_francescogabbani: 'FrankGabbani',
+  oldtownroadremix_lilnasx: 'LilNasX',
+  starships_nickiminaj: 'NickiMinaj',
+  sucker_jonasbrothers: 'JonasBrothers',
```
Comment (Erin007) [inline apps/src/code-studio/dancePartySongArtistTags.js:48]: Are any of the potentials listed below still options? Should we delete them?

Comment (epeach) [reply apps/src/code-studio/dancePartySongArtistTags.js:48]: We actually ended up getting one of last year's potentials this year, FWIW. Not sure if any of these are still in consideration. I'll check with Min

**Review by Erin007 — APPROVED** [review]

File: apps/src/code-studio/dancePartySongArtistTags.js (hunk 1 above)
Comment (Erin007) [reply apps/src/code-studio/dancePartySongArtistTags.js:48]: ok cool.  there's no real harm in keeping them, mostly just curious.

**Comment by ajpal** [comment]
Cross-checking to twitter the differences I see are:
pedrocapo
katyperry
frankgabbani
jonasbrothers

**Comment by epeach** [comment]
It looks like historically for all uncapitalized names that we are capitalized the first letter of first/last names, so I kept with that pattern. I think we should move forward with that style choice.

**Review by ajpal — APPROVED** [review]

**Pull request merged**